### DISCUSSION TITLE
浏览器选项卡上的网站图标如何配置

### DISCUSSION THREAD

**Discussion by raysun0501** [vuepress-theme-hope/vuepress-theme-hope#3245] [Q&A] [answered]
请问浏览器选项卡上的网站图标如何配置？
<img width="227" alt="image" src="https://github.com/vuepress-theme-hope/vuepress-theme-hope/assets/42628912/01a9c35d-7c31-4659-888a-248e295862eb">
<img width="330" alt="image" src="https://github.com/vuepress-theme-hope/vuepress-theme-hope/assets/42628912/d476308a-15d6-40c8-b2ba-40c714860a15">

**Comment by Mister-Hope** [comment, marked as answer]
favicon

**Reply by raysun0501** [reply]
好的，收到，thanks
